Source/missiles.cpp

```diff
@@ -3187,6 +3188,10 @@ void ProcessLightningWallControl(Missile &missile)
 	int lvl = !missile.IsTrap() ? Players[id].getCharacterLevel() : 0;
 	int dmg = 16 * (GenerateRndSum(10, 2) + lvl + 2);
 
+	// Defines the current position of the control missile moving towards var3 direction.
+	// This is used to make sure the opposite side does not create walls where walls have already been created.
+	const Point currentPosition = { missile.var1, missile.var2 };

```

Comment:
```suggestion
	// Defines the current position of the control missile moving towards left direction.
	// This is used to make sure the opposite side does not create walls where walls have already been created.
	const Point startPositionLeft = { missile.var1, missile.var2 };
```

I used `left` because in `AddFireWallControl` var3 is initialized with `Left(Left`.

Comment also applies to `ProcessFireWallControl`.

Reply:
I'm not sure `startPositionLeft` is accurate, because this would imply every position on the left side is `startPositionLeft`

Reply:
It should always be left from the perspective of the casting player position to the initial wall control position or? 🤔

Reply:
It's the current position of the vars that push left. starting position would be what var1 and var2 are when AddWallControl is called, but these get changed every frame due to ProcessWallControl.

Reply:
Anyway I did end up removing this variable in a refactor.